packages/components/src/Account/Auth/EmailAuth.tsx

```diff
@@ -101,6 +160,10 @@ export class AccountEmailAuth extends React.Component<AccountEmailAuthProps, Acc
 
     if (authResponse === "ok") {
       onEmailSend(isNewUser, emailAddress);
+      if (hasSelectedToAddToNewsletter) {
```

Comment:
is there a Jira ticket for this? Want to make sure it doesn't get lost

Reply:
Good call, I created CIVIL-381 and added a note in the code.